docs/api-reference/event-manager.md

```diff
@@ -50,11 +55,86 @@ Register an event handler function to be called on `event`.
 * `[handler]` {Function} - The function to be called on `event`.
 
 
+
 ## Supported Events and Gestures
 
-TBA
+As of the time of this writing, the following event and gesture names are supported. Refer to [`constants.js`](../../src/controls/events/event-manager.js) for the most current support.
```

Comment:
I think we can drop this, we should just keep it up to date in PR reviews, that is how we handle other code.